libraries/postgres/src/util.js

```diff
@@ -2,6 +2,16 @@ const fs = require('fs');
 const path = require('path');
 const {SYNTAX_ERROR} = require('./constants');
 
+exports.ignorePgErrors = async (promise, ...codes) => {
```

Comment:
this function returning the result on success and nothing on ignored failures is a bit weird to me. Could it make sense to not return anything in either case since nobody should use the results from this anyway without a check of what the results were?

Reply:
Good call.  #2753.